### ISSUE TITLE
PostsPerPageSniff.php does not catch the `-1` if the line is not followed by a comma

### ISSUE THREAD

**Issue by david-binda** [Automattic/VIP-Coding-Standards#29]
The sniff catches following:

```
<?php

$args = array(
    'posts_per_page' => -1,
);
```

but not following:

```
<?php

$args = array(
    'posts_per_page' => -1
);
```

**Comment by GaryJones** [comment]
Closing in favour of https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/1505.
